packages/metal-state/src/Config.js

```diff
@@ -110,12 +110,82 @@ function mergeConfig(context, config) {
 	return obj;
 }
 
-// Add all validators to `Config`.
-const fnNames = Object.keys(validators);
-fnNames.forEach(
-	name => Config[name] = function() {
+/**
+ * Adds primitive type validators to the config object.
+ * @param {string} name The name of the validator.
+ */
+function setPrimitiveValidators(name) {
+	Config[name] = function() {
 		return this.validator(validators[name]);
-	}
-);
+	};
+}
+
+setPrimitiveValidators('any');
```

Comment:
Regardless this proposal the end behavior should be the same, correct? If so this pull looks good to me and then if you want you can send a refactor for the internals.